src/app/components/home/home.component.ts

```diff
@@ -3,10 +3,8 @@ import { Component } from '@angular/core';
 @Component({
```

Comment:
what's the purpose of this component?

Reply:
this is a start page of the application which contains albums and categories components